src/app/home/home.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="wrapper">
-	<img src="{{headerImageUrl}}" alt="Loklak Cow" width="auto" height="120">
+	<img src="{{headerImageUrl}}" alt="loklak Search - Distributed Social Media Message Search Engine" title="loklak Search - Distributed Social Media Message Search Engine" width="auto" height="120">
```

Comment:
The height is actually controlled by the css [here](https://github.com/fossasia/loklak_search/pull/319/files/90144e9d7eefa7df0f2c48add647cbdb3c4c484c#diff-ca473606fcbb131b1cdc73d859819ca4R18). This is just a fallback till the css loads. The height does not need to be specified as it is automatically calculated according to aspect-ratio of image and hence does not has any chance for distortion or stretching.